src/Tribe/Tickets_Handler.php

```diff
@@ -1044,7 +1053,7 @@ public function save_global_stock( $post_id ) {
 	 *
 	 */
 	public function save_tickets_order( $post_id ) {
-		if ( ! ( isset( $_POST[ 'tribe-tickets-post-settings' ] ) && wp_verify_nonce( $_POST[ 'tribe-tickets-post-settings' ], 'tribe-tickets-meta-box' ) ) ) {
```

Comment:
I can add a nonce, but this is triggered by `save_post` on the event - we've already checked the save nonce. _But_ this is a public function. 

Think I'll just go back to checking the metabox nonce - it's already there. I misspoke earlier as to it being specifically for the settings form. We use it in a few places.